Source/control.cpp

```diff
@@ -889,19 +890,19 @@ void DrawInfoBox(const Surface &out)
 					PrintMonstHistory(monster.MType->mtype);
 				}
 			} else if (pcursitem == -1) {
-				CopyUtf8(infostr, Towners[pcursmonst].name, sizeof(infostr));
+				InfoString = std::string(Towners[pcursmonst].name);

```

Comment:
The move constructor should work without the explicit constructor call, is this failing on GCC6?

Reply:
yes :(